### PR TITLE
terraform for batch converter

### PR DESCRIPTION
**Pull request by parthshahva**
1) Vpc terraform setup
2) Converter batch terraform setup
3) Variable syntax cleanup
4) Addition to terraform Makefiles of import commands

### CODE REVIEW

File: terraform/modules/matrix-service/infra/converter_batch.tf [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+  name = "dcp-matrix-converter-job-${var.deployment_stage}"
+  policy = <<POLICY
+{
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Sid": "s3",
+        "Effect": "Allow",
+        "Action": [
+          "s3:*"
```
Comment (calvinnhieu) [inline terraform/modules/matrix-service/infra/converter_batch.tf]: Do we want to restrict this?

**Comment by calvinnhieu** [comment]
Do you want to PR this into `develop`?

File: terraform/modules/matrix-service/infra/converter_batch.tf (hunk 1 above)
Comment (mckinsel) [reply terraform/modules/matrix-service/infra/converter_batch.tf]: I think it should just need `s3:PutObject` if `s3:ListBucket` and `s3:GetObject` are open to everyone.

File: terraform/modules/matrix-service/infra/converter_batch.tf [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+  policy_arn = "${aws_iam_policy.converter_job_policy.arn}"
+}
+
+resource "aws_batch_job_definition" "converter_job_def" {
+    name = "dcp-matrix-converter-job-definition-${var.deployment_stage}"
+    type = "container"
+    container_properties = <<CONTAINER_PROPERTIES
+{
+    "command": [],
+    "image": "humancellatlas/matrix-converter:latest",
```
Comment (mckinsel) [inline terraform/modules/matrix-service/infra/converter_batch.tf]: Maybe better if I tag an image?

Comment (parthshahva) [reply terraform/modules/matrix-service/infra/converter_batch.tf]: @mckinsel I'm actually using slightly different code/image to get it to work with batch. That WIP PR is coming up shortly. I'll be deploying the image to humancellatlas/matrix-converter in dockerhub. I'm including a concept of VERSION. I agree. I'll replace latest with version here.

File: terraform/modules/matrix-service/infra/converter_batch.tf (hunk 1 above)
Comment (parthshahva) [reply terraform/modules/matrix-service/infra/converter_batch.tf]: sounds good

File: terraform/envs/staging/terraform.tfvars.example [hunk 3]
```diff
@@ -1 +1,4 @@
 deployment_stage = ""
+account_id = ""
+aws_region = ""
+converter_cluster_ami_id=""
```
[the remark this answers is not in the document]
Comment (parthshahva) [reply terraform/envs/staging/terraform.tfvars.example]: fixed

**Pull request closed without merging**